.gitignore

```diff
@@ -1,5 +1,6 @@
 /.bundle/
 /.yardoc
+/Gemfile.lock
```

Comment:
This change should not be needed as it's already on my branch: https://github.com/ainformatico/mailinator/blob/v1-release/.gitignore#L3